src/services/nexboard/services/Project.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable class-methods-use-this */
+const rpn = require('request-promise-native');
+const nexboardClient = require('../utils/Nexboard');
+
+const EDITOR_MS_URI = process.env.EDITOR_MS_URI || 'http://localhost:4001';
+const REQUEST_TIMEOUT = process.env.NODE_ENV !== 'production' ? 120 * 1000 : 6 * 1000;
+
+class Project {
+	constructor(options) {
+		this.options = options || {};
+		this.docs = {};
+	}
+
+	async get(id, params) { // id equals lessonId
+		const nexboardAttachments = await rpn({
+			uri: `${EDITOR_MS_URI}/attachments`,
```

Comment:
It should go over the way api nextboard -> editor client <-> api attachments <->editor MS but not api nextboard <-> editor MS.

Reply:
I don't really understand...